src/components/PluginLabelTagCloud.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {useStaticQuery, graphql} from 'gatsby';
+import {TagCloud} from 'react-tagcloud';
+
+function PluginLabelTagCloud() {
+
+    const allLabels = useStaticQuery(graphql`
+        query {
+            labels: allJenkinsPlugin {
+                nodes {
+                  labels
+                }
+            }
+        }
+    `).labels.nodes.reduce((prev, nodes) => {
+
+        nodes.labels.reduce((prevLabel, obj) => {
+            if (!prev[obj]) {
+                prev[obj] = 1;
+            } else {
+                prev[obj]++;
+            }
+            return obj;
+
+        }, {});
+
+        return prev;
+    }, {});
+
+    const tags = new Array();
+    Object.entries(allLabels).forEach(([value, count]) => {
```

Comment:
It wasn't obvious to me how to create this structure while doing the `reduce` ops above....

Reply:
Maybe it can be simplified by converting the inner `reduce` to `forEach` (you don't really need `prevLabel` for anything)

Reply:
i think this is fine, but you could map it

```
const tags = Object.entries(allLabels).map(([value, count]) => { return { value, count } });
```